src/main/java/pl/futurecollars/invoicing/config/Config.java

```diff
@@ -0,0 +1,7 @@
+package pl.futurecollars.invoicing.config;
+
+public class Config {
+
+  public static final String DATABASE_FILE_LOCATION = "db/invoices.txt";
```

Comment:
Jeśli w pliku będziesz przechowywał dane w formacie json, to dajmy temu plikowi odpowiednie rozszerzenie (json).